code/modelCuration/GlycolysisCuration.m

```diff
@@ -0,0 +1,94 @@
+% This function is for adding new rxns annotated for glycolysis genes of GPRs identified by GPT.
+rxnStruct = importTsvFile('../../data/modelCuration/addRxnGly_20230414.tsv');
+MetStruct = importTsvFile('../../data/modelCuration/addMetGly_20230414.tsv');
+
+model = importYaml('Human-GEM.yml');
+
+rxnsToAdd.rxns = rxnStruct.rxnID;
+rxnsToAdd.eccodes = rxnStruct.ECNumber;
+rxnsToAdd.equations = rxnStruct.Equation;
+rxnsToAdd.rxnNames = rxnStruct.rxnID;
+rxnsToAdd.lb = zeros(length(rxnStruct.rxnID),1);
+rxnsToAdd.ub = repmat(1000,length(rxnStruct.rxnID),1);
+idx = find(cell2mat(rxnStruct.Reversibility) == 1);
+rxnsToAdd.lb(idx) = -1000;
+rxnsToAdd.grRules = rxnStruct.grRules;
+rxnsToAdd.subSystems = rxnStruct.Subsystems;
+rxnsToAdd.rxnConfidenceScores = zeros(length(rxnStruct.rxnID),1);
+[~,idx] = ismember(MetStruct.MetID,model.mets);
+metsToAdd.mets = MetStruct.MetID(~idx,1);
+metsToAdd.metNames = MetStruct.Name(~idx,1);
+metsToAdd.compartments = MetStruct.Comp(~idx,1);
+metsToAdd.metFormulas = MetStruct.Fromula(~idx,1);
+metsToAdd.metCharges = MetStruct.Charge(~idx,1);
+tmp = [];
+for i = 1:length(metsToAdd.metCharges)
+    if ~isempty(metsToAdd.metCharges{i})
+        tmp(i,1) = str2double(metsToAdd.metCharges{i});
+    else
+        tmp(i,1) = NaN;
+    end
+end
+metsToAdd.metCharges = tmp;
+
+[newModel, modelChanges] = addMetabolicNetwork(model,rxnsToAdd,metsToAdd);
+
+% export the new mets, new reactions to the table
+structure = importTsvFile('../../model/metabolites.tsv');
+[~,idx] = ismember(modelChanges.mets.mets,MetStruct.MetID);
+MetStruct.BiGGID(1:length(MetStruct.MetID),1) = {''};
+% metID metNames	metFormulas	metCharges	compartments	metKEGGID	metPubChemID	metChEBIID	metMetaCycID	metMetaNetXID
+metNocomp = cellfun(@(s) s(1:8), MetStruct.MetID(idx,1), 'UniformOutput', false);
+structure.mets = [structure.mets;MetStruct.MetID(idx,1)];
+structure.metBiGGID = [structure.metBiGGID;MetStruct.BiGGID(idx,1)];
+structure.metKEGGID = [structure.metKEGGID;MetStruct.KEGG(idx,1)];
+structure.metsNoComp = [structure.metsNoComp;metNocomp];
+structure.metHMDBID = [structure.metHMDBID;MetStruct.BiGGID(idx,1)];
+structure.metChEBIID = [structure.metChEBIID;MetStruct.CHEBI(idx,1)];
+structure.metPubChemID = [structure.metPubChemID;MetStruct.BiGGID(idx,1)];
+structure.metLipidMapsID = [structure.metLipidMapsID;MetStruct.CHEBI(idx,1)];
```

Comment:
this line probably should be modified, now ChEBI ids were also updated to the column of Lipidmap ids

Reply:
They are all empty. but it is better to rename it to not cause misunderstanding.

Reply:
Thanks for identifying this. Changed in the latest commit.